gnosis/eth/tests/test_oracles.py

```diff
@@ -245,6 +246,38 @@ def test_get_price(self):
             cream_oracle.get_price(Account.create().address)
 
 
+class TestSuperfluidOracle(EthereumTestCaseMixin, TestCase):
+    def test_get_price(self):
+        ethereum_client_polygon = EthereumClient("https://polygon-rpc.com")
```

Comment:
I don't like hardcoding nodes here as it will add more dependencies and point of failures during testing, is there any chance that you could use mainnet as the other end to end tests?

Reply:
Tokens not on mainnet, query will fail.

Reply:
Ok, then please implement something like this for Polygon using an environment variable `ETHEREUM_POLYGON_NODE: https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/history/tests/utils.py#L7

Leave out Arbitrum as it's too many dependencies and if it works for Polygon it should work for Arbitrum